util/topsql/topsql.go

```diff
@@ -48,8 +49,9 @@ func SetupTopSQL() {
 
 	globalTopSQLReport = remoteReporter
 
-	tracecpu.GlobalSQLCPUProfiler.SetCollector(remoteReporter)
-	tracecpu.GlobalSQLCPUProfiler.Run()
+	cpuprofile.GlobalCPUProfiler.Start()
```

Comment:
If the profiler only starts once, I don't think it's topsql's business.

Reply:
done.